test/test_filesystem_helper.cpp

```diff
@@ -70,9 +70,27 @@ TEST(TestFilesystemHelper, join_path)
 
 TEST(TestFilesystemHelper, parent_path)
 {
-  auto p = path("my") / path("path");
-
-  EXPECT_EQ(p.parent_path().string(), path("my").string());
+  {
+    auto p = path("my") / path("path");
+    EXPECT_EQ(p.parent_path().string(), path("my").string());
+  }
+  {
+    auto p = path("foo");
+    EXPECT_EQ(p.parent_path().string(), ".");
+  }
+  {
+    if (is_win32) {
+      auto p = path("\\foo");
```

Comment:
This test is wrong according to the logic in `is_absolute()`. The prefix should have a single character followed by a colon. e.g. `C:\\`

Reply:
Fixed in 2f99349